### PR TITLE
1.1.1.1: add operators page

### PR DESCRIPTION
**Pull request by elithrar**
Summary:

* Adds a page for Network Operators using 1.1.1.1
* Re-orders the side-bar to accommodate

### CODE REVIEW

**Comment by elithrar** [comment]
cc/ @marciocloudflare

File: products/1.1.1.1/src/content/network-operators.md [hunk 1]
```diff
@@ -0,0 +1,57 @@
+---
+order: 8
+pcx-content-type: reference
+---
+
+# Network Operators
+
+Network operators, including Internet Service Providers (ISPs), device manufacturers, public WiFi networks, municipal broadband providers, and security scanning services can use 1.1.1.1 and [1.1.1.1 for Families](https://developers.cloudflare.com/1.1.1.1/1.1.1.1-for-families) in place of operating their own recursive DNS infrastructure.
```
Comment (marciocloudflare) [inline products/1.1.1.1/src/content/network-operators.md]:
```suggestion
Network operators, including Internet Service Providers (ISPs), device manufacturers, public WiFi networks, municipal broadband providers, and security scanning services can use [1.1.1.1](/setup-1.1.1.1) and [1.1.1.1 for Families](/1.1.1.1-for-families) in place of operating their own recursive DNS infrastructure.
```

File: products/1.1.1.1/src/content/network-operators.md [hunk 2]
```diff
@@ -0,0 +1,57 @@
...
+
+Using 1.1.1.1 can improve performance for end-users due to Cloudflare's extensive [global network](https://www.cloudflare.com/network/), as well as higher overall cache hit rates due to our regional caches.
+
+The 1.1.1.1 resolver was designed with a privacy-first approach. Refer to our [data and privacy policies](https://developers.cloudflare.com/1.1.1.1/privacy/public-dns-resolver) for what is logged and retained by 1.1.1.1.
```
Comment (marciocloudflare) [inline products/1.1.1.1/src/content/network-operators.md]:
```suggestion
The 1.1.1.1 resolver was designed with a privacy-first approach. Refer to our [data and privacy policies](/privacy/public-dns-resolver) for what is logged and retained by 1.1.1.1.
```

File: products/1.1.1.1/src/content/network-operators.md [hunk 3]
```diff
@@ -0,0 +1,57 @@
...
+
+## Configuring 1.1.1.1
+
+There are multiple ways to use 1.1.1.1 as an operator:
+
+* Including a [DNS over HTTPS](https://developers.cloudflare.com/1.1.1.1/encrypted-dns/dns-over-https) or [DNS over TLS](https://developers.cloudflare.com/1.1.1.1/encrypted-dns/dns-over-tls) proxy on end-user routers or devices (best for privacy)
+* Pushing 1.1.1.1 to devices via DHCP/PPP within an operator network (recommended; most practical)
+* Having a DNS proxy on a edge router make requests to 1.1.1.1 on behalf of all connected devices
```
Comment (marciocloudflare) [inline products/1.1.1.1/src/content/network-operators.md]:
```suggestion
* Including a [DNS over HTTPS](/encrypted-dns/dns-over-https) or [DNS over TLS](/encrypted-dns/dns-over-tls) proxy on end-user routers or devices (best for privacy).
* Pushing 1.1.1.1 to devices via DHCP/PPP within an operator network (recommended; most practical).
* Having a DNS proxy on a edge router make requests to 1.1.1.1 on behalf of all connected devices.
```

File: products/1.1.1.1/src/content/network-operators.md [hunk 4]
```diff
@@ -0,0 +1,57 @@
...
+
+Resolver                           | IP Addresses       | DNS over HTTPS endpoint                         | DNS over TLS endpoint
+-----------------------------------|--------------------|-------------------------------------------------|-------------------------------
+1.1.1.1 (unfiltered)               | 1.1.1.1 / 1.0.0.1  | `https://cloudflare-dns.com/dns-query`          | `cloudflare-dns.com`
+Families (Malware)                 | 1.1.1.2 / 1.0.0.2  | `https://security.cloudflare-dns.com/dns-query` | `security.cloudflare-dns.com`
+Families (Adult Content + Malware) | 1.1.1.3 / 1.0.0.3  | `https://family.cloudflare-dns.com/dns-query`   | `family.cloudflare-dns.com`
+
+</TableWrap>
+
+You may wish to provide end-users options to change from the default 1.1.1.1 resolver to one of the [1.1.1.1 for Families](https://developers.cloudflare.com/1.1.1.1/1.1.1.1-for-families) endpoints.
```
Comment (marciocloudflare) [inline products/1.1.1.1/src/content/network-operators.md]:
```suggestion
You may wish to provide end-users options to change from the default 1.1.1.1 resolver to one of the [/1.1.1.1-for-families) endpoints.
```

File: products/1.1.1.1/src/content/network-operators.md [hunk 5]
```diff
@@ -0,0 +1,57 @@
...
+## Rate Limiting
+
+Operators using 1.1.1.1 for typical Internet-facing applications and/or users should not encounter any rate limiting for their users. In some rare cases, security scanning use-cases or proxied traffic may be rate limited to protect our infrastructure as well as upstream DNS infrastructure from potential abuse. 
+
+Best practices include:
+
+* Avoiding tunnelling or proxying all queries from a single IP address at high rates. Distributing queries across multiple public IPs will improve this without impacting cache hit rates (caches are regional).
+* A high rate of "uncacheable" responses (such as `SERVFAIL`) against the same domain may be rate limited to protect upstream, authoritative nameservers. Many authoriative nameservers enforce their own rate limits, and we strive to avoid overloading third party infrastructure where possible.
+
+If you're a network operator and still have outstanding questions, contact `resolver@cloudflare.com` with your use-case and we'll be happy to discuss further.
```
Comment (marciocloudflare) [inline products/1.1.1.1/src/content/network-operators.md:57]: Do we really want to include this email address here?

**Review by marciocloudflare — CHANGES_REQUESTED** [review]

**Comment by marciocloudflare** [comment]
Hi @elithrar. Could you change the hyperlinks please? Thanks!

**Review by marciocloudflare — APPROVED** [review]

**Comment by elithrar** [comment]
Thanks @marciocloudflare!

**Comment by marciocloudflare** [comment]
Sure thing :)

**Pull request merged**